inst/example_workflow.R

```diff
@@ -0,0 +1,55 @@
+#file with example workflow for basic package functionality - will be helpful later tu build vignettes
+
+git_stats <- GitStats$new()
+
+git_stats$set_connection(
+  api_url = "https://api.github.com",
+  token = Sys.getenv("GITHUB_PAT"),
+  owners_groups = c("openpharma", "r-world-devs")
+)
+
+git_stats
+
+# examples for getting repos
+
+git_stats$get_repos_by_owner_or_group()
+
+git_stats$get_repos_by_codephrase("Dashboard")
```

Comment:
running this line raises an error on my side:
`Error in `resp_check_status()`:
! HTTP 403 Forbidden.
• You have exceeded a secondary rate limit. Please wait a few minutes before you try again.
• See docs at <https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#secondary-rate-limits>`

Reply:
Yes, I need to fix searching requests, I set up an issue for that in #32